teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Great job making the required and optional changes, unfortunately, you removed `specialization` from the parameters of the initialize method, please add it again.

```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
end
```